### ISSUE TITLE
Allow apply two sides at time

### ISSUE THREAD

**Issue by jabiertxof** [jnweiger/inkscape-exponential-distort#1]
Hi coul be great add two sides at time

**Comment by jnweiger** [comment]
Yeah, have the effect on X and Y axis for a double logarithmic scale.
I'd rather keep the code simple, until the math is settled.
Current workaround
* apply this extension
* rotate 90° 
* apply this extension again,
* rotate back.
